client/src/components/Navbar/Navbar.tsx

```diff
@@ -114,7 +114,13 @@ const Navbar: React.FC = () => {
   const renderMenuItems = () => {
     // TODO: conditionally render based on profile type
     return menuItems.map((menu) => {
-      if (menu.authenticated) {
+      if (location.pathname === '/login' && menu.authenticated === true) {
```

Comment:
Why are we checking for authenticated menu on login/signup pages? :)

Reply:
Because if I remove these statement, then all of the navbar items shows at the top of the signup and login pages.